source/hic_hal/stm32/stm32h743ii/extended_features/include/adapter_detector.h

```diff
@@ -0,0 +1,21 @@
+#ifndef _ADAPTER_DETECTOR_H_INCLUDED_
+#define _ADAPTER_DETECTOR_H_INCLUDED_
+
+#include <stdint.h>
+
+typedef enum
+{
+    ADAPTER_UDB_13_FLEX,
```

Comment:
One more suggestion on the adapter type enum - I suggest explicitly listing the value of each enum member so that one can easily look at the enum and know which number corresponds to which type.

Reply:
Sorry, also meant to add an example of what I meant:

    typedef enum
    {
        ADAPTER_UDB_13_FLEX = 0,
        ADAPTER_JOKER_FLEX  = 1,
        ...
    };